uber/site_sections/preregistration.py

```diff
@@ -506,6 +506,7 @@ def credit_card_retry(self):
         return {}
 
     # TODO: figure out if this is the best way to handle the issue of people not getting shirts
+    # TODO: this may be all now-dead one-time code (attendee.owed_shirt doesn't exist anymore)
```

Comment:
Hmm, it's possible that this code hasn't been updated to how we do things these days.  The backstory is that we still DO have a ``NoShirt`` model, and we do have the ability to turn on a button for the Merch folks that allows them to say, "This attendee did not receive a shirt with the rest of their swag" so that we can record that and direct them here for future reference.

We've turned on that button at the past couple of events (MagClassic and MAGFest 2016), in both cases after we got close enough to runnig out of stock for a couple of sizes that Ryan asked me to turn it on just in case.  However, to my recollection, we didn't end up needing it, or possibly it just got clicked so few times that we handled those cases manually.  Either way, this form hasn't been used since MAGFest 12 when we ran out of the green Zelda shirts.

In theory it's still a valid page that we might need in the future, though in practice it might be currently broken and thus would need to be fixed before we could actually turn it back on.